packages/coffeescript/plugin/compile-coffeescript.js

```diff
@@ -67,6 +68,13 @@ export class CoffeeCompiler extends CachingCompiler {
       return null;
     }
 
+    if (source.indexOf('`') !== -1) {
+      // If source contains backticks, pass the coffee output through ecmascript
+      try {
+        output.js = ECMAScript.compileForShell(output.js);
+      } catch (e) {}
```

Comment:
BTW I can't remember why I put this try/catch block here, and especially why I ignore any exceptions. Maybe I was controlling for if `ecmascript` wasn't added to a project, before I realized that I could force its presence via `api.use`. Please let me know if `ecmascript` is likely to ever throw an exception (and if so, how we can trigger that so as to write a test for it); and in general if you think this is okay.